tests/functional/test_create.py

```diff
@@ -0,0 +1,267 @@
+import os
+import sys
+from itertools import chain
+from itertools import product
+
+import pytest
+import scripttest
+
+IS_WINDOWS = (
+    sys.platform.startswith("win") or
+    (sys.platform == "cli" and os.name == "nt")
+)
+IS_26 = sys.version_info[:2] == (2, 6)
+IS_PYPY = hasattr(sys, 'pypy_version_info')
+
+
+def locate_on_path(binary):
+    paths = os.environ["PATH"].split(os.path.pathsep)
+    for path in paths:
+        binpath = os.path.join(path, binary)
+        if os.path.exists(binpath):
+            return binpath
+
+
+def resolve_path(path):
+    if path is not None:
+        path = os.path.realpath(os.path.expanduser(path))
+    return path
+
+
+PYTHON_BINS = [
```

Comment:
I would add a comment of what each of these value are. Like a header for an array. The second and third are obisou, not the first.